src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/ProjectPackages.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using NuGet.Packaging;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    /// <summary>
+    /// Struct containing installed and transitive packages for a project
+    /// </summary>
+    public struct ProjectPackages
+    {
+        public IReadOnlyList<PackageReference> InstalledPackages { get; }
+        public IReadOnlyList<PackageReference> TransitivePackages { get; }
```

Comment:
```C#
        public IReadOnlyList<PackageReference> InstalledPackages { get; } = Array.Empty<PackageReference>();
        public IReadOnlyList<PackageReference> TransitivePackages { get; } = Array.Empty<PackageReference>();
```

Reply:
I tried to make this change, but get the error 'ProjectPackages' : cannot have instance property or field initializers in structs